pom.xml

```diff
@@ -29,9 +30,9 @@
     <artifactId>forgerock-parent</artifactId>
     <version>2.0.7</version>
```

Comment:
i understand how you feel. however, the way I see it is two-fold:
- priority #1 is to get everything building without having to use our "copy to local repo" workaround, then we can make changes and put out proper releases.
- i think it's acceptable to put out a modified version as long as the only changes are to the repo and artifact human-readable name. semver would say that the version should indicate compatibility, and the changes we're making don't affect how the artifact works with other artifacts -- it's opposite. we're trying to provide artifacts that are binary equivalents / drop-ins for the original artifacts that required FR's original repo.